io/src/main/scala/com/velocidi/apso/io/SftpFileDescriptor.scala

```diff
@@ -240,6 +240,8 @@ object SftpFileDescriptor {
     }
   }
 
+  case class SftpFileDescriptorCredentials(host: String, port: String, auth: Either[Identity, String])
```

Comment:
Since this is already scoped under the `SftpFileDescriptor` object, I think we can remove the prefix and just go with `Credentials`. We can use `SftpFileDescriptor.Credentials` for extra clarity.

Reply:
Agreed! Updated 👍